### PR TITLE
Логика разделения товарных позиций

### PR DESCRIPTION
**Pull request by Mavlyan**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,6 @@ before_script:
 script:
   - php vendor/bin/parallel-lint --exclude vendor .
   - php vendor/bin/phpcs --standard=vendor/magento-ecg/coding-standard/Ecg,vendor/mygento/coding-standard/Mygento-Mage1 --ignore=lib .
-  - php vendor/bin/phpcpd --exclude=vendor .
+  - php vendor/bin/phpcpd --exclude=vendor --exclude tests/ .
```
Comment (luckyraul) [inline .travis.yml]: remove /

File: app/code/community/Mygento/Kkm/Helper/Discount.php [hunk 2]
```diff
@@ -220,6 +220,69 @@ protected function _buildItem($item, $price, $taxValue = '')
...
+            $final[$item->getId()] = $item1;
+
+            return $final;
+        }
+
+        $item2['price'] = $item2['price'] + 0.01 + $inc / 100;
+        $item2['quantity'] = $qtyUpdate;
+        $item2['sum'] = round($item2['quantity'] * $item2['price'], 2);
+
+        $final[$item->getId() . '_1'] = $item2;
```
Comment (luckyraul) [inline app/code/community/Mygento/Kkm/Helper/Discount.php]: not symmetrical

**Comment by Mavlyan** [comment]
Do not merge this request. Because first we should sync split-tmp with mygento/kkm:feature-disable-internal-calculations branch

File: app/code/community/Mygento/Kkm/etc/config.xml [hunk 3]
```diff
@@ -186,6 +186,9 @@
                 <cron_update_expr>10 * * * *</cron_update_expr>
                 <autosend_limit>5</autosend_limit>
                 <force_resend>0</force_resend>
+                <apply_algorithm>0</apply_algorithm>
```
Comment (luckyraul) [inline app/code/community/Mygento/Kkm/etc/config.xml]: default 1

**Pull request merged**